lib/src/debug_logger.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter_reactive_ble/src/model/log_level.dart';
+
+class DebugLogger {
+  DebugLogger(
+    String tag,
+    void Function(Object object) print,
+  )   : _tag = tag,
+        _print = print;
+
+  set logLevel(LogLevel logLevel) => _logLevel = logLevel;
+
+  void log(Object message) {
```

Comment:
To be able to introduce new log-levels this method will need to be extended with the message-log-level parameter. But, I guess, that can wait.